### ISSUE TITLE
Swipe to unlock overlap

### ISSUE THREAD

**Issue by duraidabdul** [andrewwiik/LittleX#5]
On iPhone SE, the Swipe up to unlock text and the Camera/Flashlight toggles overlap
![img_1081](https://user-images.githubusercontent.com/15973483/35663531-f371bc0e-06da-11e8-9506-51feb398681e.PNG)

**Comment by andrewwiik** [comment]
You're putting an X on an IPhone SE lol, what did you expect, I will try to look for a fix though

**Comment by duraidabdul** [comment]
Lol I know that, but the whole point of this tweak is to port X features to older devices though isn't it? Would just be great if the toggles were shifted upwards a little. If its really difficult it's not that big of a problem though

**Comment by duraidabdul** [comment]
Yeah wasn't not expecting anything specific, just pointing it out, in case if it's an easy fix
